tests/test_cli_opts.py

```diff
@@ -63,3 +63,216 @@ def test_cli_prcellgid():
     )
     assert (test_folder_path / "62798_py_Neuron_t0.0.nrndat").is_file()
     assert (test_folder_path / "62798_py_Neuron_t100.0.nrndat").is_file()
+
+
+@pytest.mark.skipif(
+    not os.environ.get("NEURODAMUS_NEOCORTEX_ROOT"),
+    reason="Test requires loading a neocortex model to run")
```

Comment:
I would love if we could not depend on neocortex so this could go into the new `integration` tests 😢

Reply:
Me too...but I'm afraid is not possible... :S